Userland/Libraries/LibWeb/DOM/Node.h

```diff
@@ -761,7 +761,7 @@ class Node : public EventTarget {
 
     void build_accessibility_tree(AccessibilityTreeNode& parent);
 
-    ErrorOr<String> name_or_description(NameOrDescription, Document const&, HashTable<UniqueNodeID>&) const;
+    ErrorOr<String> name_or_description(NameOrDescription, Document const&, HashTable<UniqueNodeID>&, bool isDescendant) const;
```

Comment:
Snake-case C++ variable names please.

We also try to use named enums over boolean parameters, because just seeing "true" or "false" at the call site doesn't make it immediately obvious what is true or false. Something like:
```c++
enum class IsDescendant {
    No,
    Yes,
};
```